docs/planning-experiments/index.md

```diff
@@ -0,0 +1,16 @@
+# Planning Experiments
+
+Eppo's Sample Size Calculator operates on top of your data to automatically suggest how many subjects you'll need for an experiment.
+
+### Quickstart
+
+1. From the **[Definitions](https://eppo.cloud/definitions)** page, create a new **[Entry Point SQL](./setting_up_the_sample_size_calculator#creating-entry-points)**.
+2. Visit the **Sample Size Calculator** from the **[Experiments](https://eppo.cloud/experiments)** page. 
+3. Select your **Entry Point** and manually refresh its data. (Entry Point data [refreshes automatically on a weekly basis.](./setting_up_the_sample_size_calculator#refreshing-entry-point-data))
+4. Calculate the number of users you'll need for your experiment [using Eppo](./using_the_sample_size_calculator)!
```

Comment:
Nit: instead of `using Eppo`, I think you can be more specific and say the `Sample size calculator`, and maybe also add that you can find this on the experiment page?

Reply:
Done!